packages/web3-eth2-core/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "web3-eth2-core",
+    "version": "1.3.0",
+    "description": "Web3 ETH2 core tools for sub-packages. This is an internal package.",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.x/packages/web3-eth2-core",
+    "license": "LGPL-3.0",
+    "engines": {
+        "node": ">=8.0.0"
+    },
+    "types": "types/index.d.ts",
+    "scripts": {
+        "dtslint": "dtslint --localTs ../../node_modules/typescript/lib types",
```

Comment:
Do we have to enter the node_modules?

Reply:
I'm not sure, in this case I just copied from the previously existing packages (I think all current packages do this)